File: src/CrowdfundingApp.Client/src/app/models/requests/orders/AcceptorderRequestMessage.ts
export class AcceptOrderRequestMessage {
  public rewardId: string;
  public count: number;
  public surname: string;
  public name: string;
  public middleName: string;
  public countryId: string;
  public fullAddress: string;
  public postCode: string;
  public payCardNumber: string;
  public payCardOwnerName: string;
  public payCardCvv: string;
  public payCardExpirationDate: Date;
}

File: src/CrowdfindingApp.Common/Messages/Orders/AcceptOrderRequestMessage.cs

using System;

namespace CrowdfindingApp.Common.Messages.Orders
{
    public class AcceptOrderRequestMessage : MessageBase
    {
        public string RewardId { get; set; }
        public int Count { get; set; }
        public string Surname { get; set; }
        public string Name { get; set; }
        public string MiddleName { get; set; }
        public string CountryId { get; set; }
        public string FullAddress { get; set; }
        public string PostCode { get; set; }
        public string PayCardNumber { get; set; }
        public string PayCardOwnerName { get; set; }
        public string PayCardCvv { get; set; }
        public DateTime? PayCardExpirationDate { get; set; }
    }
}
